src/ResourceManager/Network/Commands.Network/help/Add-AzureRmRouteFilterRuleConfig.md

```diff
@@ -35,6 +35,8 @@ The Add-AzureRmRouteFilterRuleConfig cmdlet adds a route filter rule to an Azure
 
 
 
+
```

Comment:
@shahabhijeet, *.md files line breaks are added during the process of help autogeneration. Could you please clarify, should I manually update autogenerated files or keep them as is in this case?